devlog/_plan/260816_pr_merge_campaign/000_triage.md

```
@@ -0,0 +1,75 @@
+| #1727 | REVIEW_REQUIRED | none | awaiting approval |
+| #1728 | REVIEW_REQUIRED | `label` CANCELLED | awaiting approval |
+| #1729 | REVIEW_REQUIRED | none | awaiting approval + stacked base |
+| #1732 | REVIEW_REQUIRED | none | awaiting approval + stacked base |
+| #1740 | REVIEW_REQUIRED | none | awaiting approval |
+| #1750 | REVIEW_REQUIRED | none | awaiting approval |
+| #1764 | REVIEW_REQUIRED | none | awaiting approval |
+| #1793 | **CHANGES_REQUESTED** | `label`, `enforce-target` CANCELLED | requested changes must be resolved |
+| #1799 | REVIEW_REQUIRED | `macos` pending | **5 unresolved CodeRabbit threads** |
+
+Eight of nine are gated purely on approval. Only two carry real work:
+#1793 has a CHANGES_REQUESTED review, and #1799 (my own unit) has five
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)